recipe/build.sh

```diff
@@ -1,8 +1,18 @@
 #!/bin/bash
 
-cmake -DCMAKE_BUILD_TYPE=Release     \
-      -DCMAKE_INSTALL_PREFIX=$PREFIX \
+mkdir build
+cd build
+
+cmake ${CMAKE_ARGS} \
       $SRC_DIR/source
 
 make -j${CPU_COUNT}
 make install
+
+if [[ ${clangdev} == '5.*' ]]; then
+    echo "Making xeus-cling based Jupyter kernels"
+    mkdir -p $CONDA_PREFIX/share/jupyter/kernels/
+    cp -r $RECIPE_DIR/kernels/* $CONDA_PREFIX/share/jupyter/kernels/
```

Comment:
The recipe should install into `$PREFIX` (`$CONDA_PREFIX` could point to anything and in this case points to `$BUILD_PREFIX`):
```suggestion
    mkdir -p $PREFIX/share/jupyter/kernels/
    cp -r $RECIPE_DIR/kernels/* $PREFIX/share/jupyter/kernels/
```